src/containers/Navbar/NavbarMax.jsx

```diff
@@ -25,7 +25,7 @@ class NavbarMax extends React.Component {
           <Space size={22} />
           <Avatar {...user} size={54} />
           <Space size={8} />
-          <Typography type='username'>{user.username}</Typography>
+          <Typography id='username-sidebar-output' type='username'>{user.username}</Typography>
```

Comment:
maybe rename this id to something like nabvar-username

Reply:
ok done